src/nvim/options.lua

```diff
@@ -1323,8 +1323,9 @@ return {
       defaults = { if_true = '' },
       desc = [=[
         A template for a comment.  The "%s" in the value is replaced with the
-        comment text.  For example, C uses "/*%s*/". Currently only used to
-        add markers for folding, see |fold-marker|.
+        comment text. For example, C uses "/*%s*/". Most importantly, this
+        is used for commenting and uncommenting, see |commenting|.
+        Also used to add markers for folding, see |fold-marker|.
```

Comment:
```suggestion
        comment text. For example, C uses "/*%s*/". Used for |commenting| and to
        add markers for folding, see |fold-marker|.
```

I think being as concise as possible is usually the goal in built-in docs.